### ISSUE TITLE
pocketchip unable to initialize EGL display

### ISSUE THREAD

**Issue by scheckley** [ptitSeb/gl4es#157]
Pocketchip using Debian Jessie

compiled using:

`mkdir build; cd build; cmake .. -DCHIP=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo; make`

glxgears errors with:

```
LIBGL: Initialising gl4es
LIBGL: v1.1.3 built on Feb 27 2020 10:05:28
LIBGL: Using GLES 2.0 backend
LIBGL: loaded: libGLESv2.so
LIBGL: loaded: libEGL.so
LIBGL: Using GLES 2.0 backend
LIBGL: Targeting OpenGL 2.1
LIBGL: WARNING, No Limited or Full NPOT support in hardware, Forcing NPOT have no effect!
LIBGL: Not trying to batch small subsequent glDrawXXXX
LIBGL: try to use VBO
LIBGL: glX Will try to recycle EGL Surface
LIBGL: Current folder is:/home/chip
Error: glXCreateContext failed
LIBGL: Shuting down
```

does anybody having any advice?  Thanks.

**Comment by scheckley** [comment]
fixed - I was running linux-headers-4.4.154-cip28+, rolling back to ntc's default mainline kernel works with es2gears. glxgears outputs a blank screen, but is recording fps at the console.

**Comment by ptitSeb** [comment]
Ok, great!

You fixed it yourself :)

(but I'm not sure if it's really working, as glxgear should gives roughly the same output as es2gears).
